content/300-guides/300-upgrade-guides/01-upgrade-from-prisma-1/07-upgrading-a-rest-api.mdx

```diff
@@ -126,7 +126,7 @@ Consider each occurrence of the Prisma client instance `prisma` and replacing wi
 
 ### 1.1. Adjusting the import
 
-Since Prisma Client 2 is generated into `node_modules/@prisma/client`, it is imported as follows:
+Since Prisma Client 2 is generated into `node_modules/.prisma/client`, it is imported as follows:
```

Comment:
This sentence makes no sense this way any more - this now requires additional explanation.